src/uct/ib/dc/accel/dc_mlx5.c

```diff
@@ -960,6 +960,7 @@ static uct_dc_iface_ops_t uct_dc_mlx5_iface_ops = {
     },
     .fc_ctrl                  = uct_dc_mlx5_ep_fc_ctrl,
     .fc_handler               = uct_dc_iface_fc_handler,
+    .destroy_qp               = uct_rc_iface_qp_destroy
```

Comment:
why is mlx5_qp_destroy not relevant for dc_x?

Reply:
because for DC, there is no receive side, so there is no situation when destroying the QP generates SRQ-related CQEs